test/integration/pack/test_pack.py

```diff
@@ -696,6 +698,87 @@ def test_project_without_stateboard(cartridge_cmd, project_without_dependencies,
     assert '{}-stateboard.service'.format(project.name) not in systemd_files
 
 
+@pytest.mark.parametrize('pack_format', ['rpm', 'deb', 'tgz', 'docker'])
```

Comment:
Let's reduce the number of tests. In fact, building the project is common for all types. Such common parts are tested only for `tgz` (see `test_packing_without_git` for example). It allows to reduce time of running tests.

Reply:
You can also add second paramtrization parameter for local build and build in docker. See `test_app_with_non_existing_hook` test.